conduit/data/datasets/utils.py

```diff
@@ -125,7 +125,9 @@ def img_to_tensor(img: Image.Image | np.ndarray) -> Tensor:
 
 def infer_al_backend() -> AudioLoadingBackend:
     """Infer which audio-loading backend to use based on the operating system."""
-    return 'soundfile' if platform.system() == 'Windows' else 'sox_io'
+    soundfile: Literal["soundfile"] = "soundfile"
+    sox: Literal["sox_io"] = "sox_io"
```

Comment:
This should also work to make the IDE see them as `Literal` (at least this works in mypy)

```suggestion
    soundfile: Final = "soundfile"
    sox: Final = "sox_io"
```